config-model/src/test/java/com/yahoo/searchdefinition/processing/ImportedFieldsTestCase.java

```diff
@@ -65,6 +69,173 @@ private static Search buildAdSearch(String sdContent) throws ParseException {
         return builder.getSearch("ad");
     }
 
+    private static void check_struct_import(ParentSdBuilder parentBuilder) throws ParseException {
```

Comment:
Please rename to checkStructImport. Underscores are mainly for test function names.